client/src/interfaces/eventData.tsx

```diff
@@ -0,0 +1,10 @@
+export default interface eventData {
+  name: string;
+  description: string;
+  img: string;
+  status: 'in-progress' | 'closed' | 'upcoming';
+  startTime: string;
+  profileImage: string;
+  username: string;
+// eslint-disable-next-line semi, no-extra-semi
```

Comment:
to fix the `no-extra-semi` problem 
add `"semi": [1, "never"]` to the eslint `rules` in `.eslintrc.json` for the client